src/app/config/config.c

```diff
@@ -517,6 +517,8 @@ static const config_var_t option_vars_[] = {
       LINELIST_S, RendConfigLines, NULL),
   VAR("HiddenServiceEnableIntroDoSBurstPerSec",
       LINELIST_S, RendConfigLines, NULL),
+  VAR("HiddenServiceOnionBalanceConfigFile",
```

Comment:
Design question: why is this a separate configuration file, instead of a set of options for use with the service?

Reply:
For example, it seems like it would make more sense to have this be a file at a fixed location in the hidden service directory, maybe with a boolean in the torrc to turn it on.